meta.xml

```diff
@@ -5,7 +5,7 @@
   <description>Check your websites for viruses automatically using multiple anti-virus engines.</description>
   <description xml:lang="ru-RU">Автоматизируйте бесплатную проверку своих сайтов на вирусы.</description>
   <version>1.0</version>
```

Comment:
Any changes in functionality should lead to increment of the version. Incrementing of release are usually happen only when meta files are modified or several attempts of certification.